src/test/scala/de/upb/cs/swt/delphi/instanceregistry/connection/ServerTest.scala

```diff
@@ -831,6 +1013,11 @@ class ServerTest
     Jwt.encode(claim, configuration.jwtSecretKey, JwtAlgorithm.HS256)
   }
 
+  private def hashString(secret: String): String = {
```

Comment:
My IDE says this private definition is unused. If it's not necessary please remove it :+1: